<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Caesium Image Compressor</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QSplitter" name="main_Splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="left_Splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QTreeView" name="imageList_TreeView">
           <property name="dragDropMode">
            <enum>QAbstractItemView::DropOnly</enum>
           </property>
           <property name="defaultDropAction">
            <enum>Qt::MoveAction</enum>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="indentation">
            <number>0</number>
           </property>
           <property name="itemsExpandable">
            <bool>false</bool>
           </property>
           <property name="sortingEnabled">
            <bool>false</bool>
           </property>
           <property name="animated">
            <bool>true</bool>
           </property>
           <property name="expandsOnDoubleClick">
            <bool>false</bool>
           </property>
           <attribute name="headerShowSortIndicator" stdset="0">
            <bool>false</bool>
           </attribute>
           <attribute name="headerStretchLastSection">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="addFiles_Button">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeFiles_Button">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>247</width>
               <height>17</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="compress_Button">
             <property name="text">
              <string>Compress</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QZoomGraphicsView" name="preview_graphicsView">
        <property name="dragMode">
         <enum>QGraphicsView::ScrollHandDrag</enum>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="sidebar_VerticalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QGroupBox" name="compressionOptions_GroupBox">
          <property name="title">
           <string>Compression</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QFrame" name="compressionSlider_Frame">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="spacing">
               <number>3</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>9</number>
              </property>
              <item>
               <widget class="QSlider" name="compression_Slider">
                <property name="maximum">
                 <number>4</number>
                </property>
                <property name="pageStep">
                 <number>1</number>
                </property>
                <property name="value">
                 <number>3</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="invertedControls">
                 <bool>false</bool>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::TicksBelow</enum>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="compressionSliderLabels_HorizontalLayout">
                <property name="spacing">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="compressionSliderSpeed_Label">
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Speed</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="compressionSliderLabels_Spacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>90</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="compressionSliderQuality_Label">
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Quality</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="lossless_Checkbox">
             <property name="text">
              <string>Lossless</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="keepMetadata_Checkbox">
             <property name="text">
              <string>Keep Metadata</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="resize_groupBox">
          <property name="title">
           <string>Resize</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="fitTo_Label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Resize to fit</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="fitTo_ComboBox">
                 <item>
                  <property name="text">
                   <string>Dimensions</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Percentage</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Short edge</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Long edge</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="width_Label">
                 <property name="text">
                  <string>Width</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="width_SpinBox">
                 <property name="suffix">
                  <string>px</string>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>99999</number>
                 </property>
                 <property name="value">
                  <number>1000</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="height_SpinBox_2">
                 <property name="text">
                  <string>Height</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="height_SpinBox">
                 <property name="suffix">
                  <string>px</string>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>99999</number>
                 </property>
                 <property name="value">
                  <number>1000</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QCheckBox" name="keepAspectRatio_CheckBox">
               <property name="text">
                <string>Keep aspect ratio</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="doNotEnlarge_CheckBox">
               <property name="text">
                <string>Do not enlarge</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="output_groupBox">
          <property name="title">
           <string>Output</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="outputFolder_Label">
             <property name="text">
              <string>Folder</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>6</number>
             </property>
             <item>
              <widget class="QLineEdit" name="outputFolder_LineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maxLength">
                <number>1024</number>
               </property>
               <property name="frame">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="outputFolderBrowse_Button">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>...</string>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="outputSuffix_Label">
             <property name="text">
              <string>Suffix</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="outputSuffix_LineEdit">
             <property name="maxLength">
              <number>128</number>
             </property>
             <property name="placeholderText">
              <string>_compressed</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="keepStructure_Checkbox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Keep folder structure</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>943</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Caesium_Image_Compressor"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd_files"/>
    <addaction name="actionAdd_folder"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionRemove"/>
    <addaction name="actionClear"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout_Caesium_Image_Compressor">
   <property name="text">
    <string>About Caesium Image Compressor</string>
   </property>
  </action>
  <action name="actionAdd_files">
   <property name="text">
    <string>Add files...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAdd_folder">
   <property name="text">
    <string>Add folder...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QZoomGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>./widgets/QZoomGraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
